templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -17,6 +17,10 @@ email address or part of the username of the user you want to @-mention.
 When you begin typing an email address or username, autocomplete suggestions
 will appear.
 
+    !!! tip ""
+        You can use `@**all**` to mention everyone subscribed to the stream you
```

Comment:
I think "you can type" would be a better word choice than "you can use" (as it suggests there is a button/feature to make it appear).